include/ordering/cuthillMckeeOrdering.hpp

```diff
@@ -121,16 +124,24 @@ class CuthillMckeeOrdering : public Pgr_messages{
       */
      std::vector <II_t_rt> get_results(
             std::vector <size_type> & inv_perm,
-            const G & graph) {
+            const G &graph) {
             std::vector <II_t_rt> results;
-
+#if 0
          typename boost::graph_traits <Graph> ::vertex_iterator v, vend;
  
          for (boost::tie(v, vend) = vertices(graph.graph); v != vend; ++v) {
              auto seq = graph[*v].id;
              auto order = inv_perm[*v];
              results.push_back({{seq}, {static_cast<int64_t>(order + 1)}});
-         }
+#endif
+        for (std::vector<Vertex>::const_iterator i = inv_perm.begin();
+             i != inv_perm.end(); ++i) {
+            log << inv_perm[*i] << " ";
+            auto seq=graph[*i].id;
+            results.push_back({{seq}, {static_cast<int64_t>(inv_perm[*i])}});
```

Comment:
Here, the boost library uses an index map to store the nodes in results. But the problem is, I didn't know how to introduce an index map of the same type that is present in the boost library to the graph type which I followed.